server/env.js

```diff
@@ -6,8 +6,7 @@ if (!process.env.NODE_ENV) {
 
 if (['production', 'circleci'].indexOf(process.env.NODE_ENV) === -1) {
   const dotenv = require('dotenv');
-  dotenv.config(); // this loads .env with real values. It needs to be first because dotenv doesn't overwrite any values
-  dotenv.config({ path: 'default.env' }); // this loads the default file in github with dummy values
+  dotenv.config({ path: `.env.${process.env.NODE_ENV}` }); // this loads the default file in github with dummy values
```

Comment:
Does the comment `// this loads the default file in github with dummy values` still stand?

Reply:
i replaced the `default.env` for either `.env.development` or `.env.test` to make it more generic.

Reply:
Ah, I understood your question now @znarf , well as far as I understood this comment still makes sense. I understood it's just saying it's going to load the `.env` file in the root of the project(that's what I think they meant with *github*).

Reply:
@marcogbarcellos 

- [ ] delete the 'default.env' if it's not used
- [ ] the dummy comment was valid for `default.env`, it's not anymore with the new behavior
- [ ] seems we're not loading `.env` anymore, that's a concern

Reply:
thanks for pointing that out @znarf . you're right, I got confused about both files but I believe they're meant to run either the project or tests locally and, in my opinion I can just merge the `default.env` file content into `.env.development`.We can see this looking the [README](https://github.com/opencollective/opencollective-api#steps) steps to run the project locally. So I'll create another PR to update the README and remove the `default.env`, This PR will have the following(if you agree with them): 
- [ ] delete the 'default.env' file
- [ ] remove the dummy comment that was previously valid for `default.env`
- [ ] Update the README with new instructions 

regarding your following concern: 
- `removing `dotenv.config();`, seems we're not loading `.env` anymore, that's a concern` - This would be necessary if we were to follow the README process of renaming the `default.env` to `.env` locally. As we will just use the `.env.development` or `.env.test`locally we won't need to load 2 dotenv files.

Can I go ahead and create another PR with the fixes? once more, thanks for pointing it out.

Reply:
@marcogbarcellos Ok, you can proceed, we will later see if we can improve further.